### PR TITLE
[kube-prometheus-stack] Bump Kube-prometheus-stack versions

### PR DESCRIPTION
**Pull request by QuentinBisson**
Signed-off-by: QuentinBisson <quentin@giantswarm.io>
Co-authored-by: desaintmartin <cedric@desaintmartin.fr>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by zanhsieh — DISMISSED** [review]

**Review by monotek — DISMISSED** [review]

**Review by zanhsieh — APPROVED** [review]

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 20.0.1
+version: 20.0.2
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:21]: imho minor version should be raised.

File: charts/kube-prometheus-stack/values.yaml [hunk 2]
```diff
@@ -2057,7 +2057,7 @@ prometheus:
     ##
     image:
       repository: quay.io/prometheus/prometheus
-      tag: v2.28.1
+      tag: v2.30.3
```
Comment (monotek) [inline charts/kube-prometheus-stack/values.yaml:2060]:
```suggestion
      tag: v2.31.1
```

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/Chart.yaml (hunk 1 above)
Comment (QuentinBisson) [reply charts/kube-prometheus-stack/Chart.yaml:21]:
```suggestion
version: 20.1.0
```

**Comment by desaintmartin** [comment]
Could you signoff the commit(s) to please DCO test?

**Comment by QuentinBisson** [comment]
Sure, I thought I did yesterday

**Review by zanhsieh — APPROVED** [review]

**Review by monotek — APPROVED** [review]

**Pull request merged**